test/EndToEndTests/Tests/Client/Build.Desktop/QueryCountTests.cs

```diff
@@ -56,6 +56,21 @@ public void CountLinqTest()
             Assert.AreEqual(response.TotalCount, 10);
         }
 
+        /// <summary>
+        /// IncludeTotalCount(true) Test
+        /// </summary>
+        [TestMethod]
+        public void CountWithBoolTrueParamLinqTest()
+        {
+            var context = this.CreateWrappedContext<DefaultContainer>().Context;
+
+            var query = context.Customer.IncludeTotalCount(true);

```

Comment:
>var query = context.Customer.IncludeTotalCount(true); [](start = 12, length = 53)

should have negative test case to test "false"

Reply:
There is one